museum/sampleRecords/subjectList.destination.json

```diff
@@ -0,0 +1,169 @@
+{
+  "ok":true,
+  "results":[
+    {
+      "occurrence_id":"1",
+      "id":"1",
+      "idno":"Anderson, ND",
+      "display_label":"Anderson, ND",
+      "ca_occurrences.type_id":"Person",
+      "ca_occurrences.idno":"Anderson, ND",
+      "ca_occurrences.preferred_labels":"Anderson, ND",
+      "ca_occurrences.LegacyID":"36",
+      "ca_occurrences.SubjectDates":"1910-1980",
+      "ca_occurrences.LastEditBy":"JILLM",
+      "ca_occurrences.LastEditDate":"1 January 2001",
+      "ca_objects.idno":"A1986.52",
+      "ca_objects.type_id":"Artwork",
```

Comment:
Just curious, how did this change from `ArtWork` in the source data to `Artwork` here?  (I prefer this spelling, but it's changing the data and I can't see where).